Tests/HummingbirdWebSocketTests/WebSocketTests.swift

```diff
@@ -162,6 +161,52 @@ final class HummingbirdWebSocketTests: XCTestCase {
         )
     }
 
+    func testClientAndServerWithRouter(
+        webSocketRouter: Router<some WebSocketRequestContext>,
+        uri: URI,
+        getClient: @escaping @Sendable (Int, Logger) throws -> WebSocketClient
+    ) async throws {
+        try await withThrowingTaskGroup(of: Void.self) { group in
+            let promise = Promise<Int>()
+            let logger = {
+                var logger = Logger(label: "WebSocketTest")
+                logger.logLevel = .debug
+                return logger
+            }()
```

Comment:
```suggestion
            var logger = Logger(label: "WebSocketTest")
            logger.logLevel = .debug
```

Reply:
That's means we have a mutable logger, which we cannot use in a child task